joomla/joomla-cms
# [4.3] Media Manager support for svg images + set natural{width/height}

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/39571, https://github.com/joomla/joomla-cms/issues/39538, https://github.com/joomla/joomla-cms/issues/39543 . Complimentary to https://github.com/joomla/joomla-cms/pull/39574 Previous attempt: https://github.com/joomla/joomla-cms/pull/38722

### Summary of Changes

- Fixes the wrong mime type for svg
- adds a load event to grab the naturalWidth/naturalHeight

### In pictures

![Screenshot 2023-01-10 at 09 18 40](https://user-images.githubusercontent.com/3889375/211498069-0db49b18-200f-4dc5-bc34-a2ab5518dc6d.png)
![Screenshot 2023-01-10 at 09 18 51](https://user-images.githubusercontent.com/3889375/211498067-82e7c69f-4f18-40b0-bf7e-73f501955ada.png)
![Screenshot 2023-01-10 at 09 19 02](https://user-images.githubusercontent.com/3889375/211498062-15a53e02-2780-4ed8-827d-a37ce9792226.png)

### Testing Instructions
- First add `svg` to the image extensions, allowed extensions and `image/svg+xml` to the allowed mime types
- Goto Caasiopeia styles and select a new logo
- Upload an svg
- Select the svg and check that the url has width/height at the end

### Actual result BEFORE applying this Pull Request



### Expected result AFTER applying this Pull Request

![Screenshot 2023-01-10 at 09 19 27](https://user-images.githubusercontent.com/3889375/211498214-85695258-1364-4dbb-a4fe-fcb00e9cb5e5.png)


### Link to documentations
Please select:
